assets/scss/_variables.scss

```diff
@@ -1,2 +1,21 @@
+// Colors
 $font-light: #fff;
+$main-color: #0D4C5E;
 $secondary-color: #b4437b;
```

Comment:
for better understanding using system with variables you can look at Bootstrap:
https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss

the thing is that you can use $white: #fff; pink, blue, etc. to know which colour it is, and then set it to the different variable but it's up to you